llvm/test/Transforms/InstCombine/fdiv-cosh-sinh.ll

```diff
@@ -0,0 +1,114 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -S -passes=instcombine < %s | FileCheck %s
```

Comment:
Probably need run lines with different triples for different libcall availability (at least needs to use an explicit triple)

Reply:
I think I understand what you want me to improve but here I am not sure what you mean with "different triples".
I dont't have a lot of time currently but I will implement it when have the chance